pkgs/by-name/su/sui/package.nix

```
@@ -0,0 +1,64 @@
+# export a function that can build various Linux Sui Versions
+{
+  lib,
+  system ? builtins.currentSystem,
```

this should be `stdenv.hostPlatform.system` (which works when cross compiling iirc)
```suggestion
  system ? stdenv.hostPlatform.system,
```

---

Thanks for the recommendation, this follows the standard I've seen other packages follow as well.
I think this should help with cross compilation too, not 100% sure.